docs/howto/account-billing/forgot-your-password.md

```diff
@@ -0,0 +1,55 @@
+# Forgot your password or username
```

Comment:
Due to our preference for the gerund form in titles, I would suggest changing the title to something like "Resetting your password or reclaiming your username".

Reply:
Have changed to Resetting your password.

I´m just wondering.

In this guide you can do both, "Resetting your password and reclaiming your username".

Isn't it easy for customers to miss that this guide also contains how they can get hold of their username if title have nothing about usernames ?

Reply:
You are absolutely right. That's why my initial suggestion for a title change mentioned **both**: "Resetting your password or reclaiming your username".

Reply:
Ahaa :-) Sorry, misunderstood

Reply:
Sorry again - Done the title change to both